src/components/widgets/InputPopup.ts

```diff
@@ -420,13 +468,32 @@ export class InputPopup extends EventEmitter {
 
         let content = ""
         // Draw an input field
-        content += `${boxChars["normal"].vertical}${"> "}${this.value}█${" ".repeat(windowWidth - this.value.toString().length - 3)}${boxChars["normal"].vertical}${EOL}`
+        // if (this.value.toString().length === 0 && this.placeholder?.length)
+        //     content += `${boxChars["normal"].vertical}${"> "}${chalk.gray(
+        //         this.placeholder
+        //     )}${" ".repeat(windowWidth - this.placeholder.length - 2)}${boxChars["normal"].vertical
+        //     }${EOL}`
+        // else
+        content += `${boxChars["normal"].vertical}${"> "}${this.value
+        }█${" ".repeat(windowWidth - this.value.toString().length - 3)}${boxChars["normal"].vertical
+        }${EOL}`
 
         const windowDesign = `${header}${content}${footer}`
         const windowDesignLines = windowDesign.split(EOL)
-        const centerScreen = Math.round((this.CM.Screen.width / 2) - (windowWidth / 2))
+        const centerScreen = Math.round(this.CM.Screen.width / 2 - windowWidth / 2)
         windowDesign.split(EOL).forEach((line, index) => {
-            this.CM.Screen.cursorTo(centerScreen + this.offsetX, this.marginTop + index + this.offsetY)
+            this.CM.Screen.cursorTo(
+                centerScreen + this.offsetX,
+                this.marginTop + index + this.offsetY
+            )
```

Comment:
I've added this to blink the cursor also when there's a value.
For the rest, it's a really nice work, thank you so much =)